packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4631,11 +4631,22 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     );
   }
 
+  /// Checks the [MediaQuery.boldTextOf] on the [context].
+  ///
+  /// This makes sure that accessibility settings related to bold text are
+  /// respected.
+  TextStyle updateStyleForBold() {
```

Comment:
should the update function be called in didChangeDependencies and cache the value?

Reply:
also this should probably be private

Reply:
Deleted this method and just inlined it into `didChangeDependencies` and `didUpdateWidget`.